### ISSUE TITLE
Organisation

### ISSUE THREAD

**Issue by IntelOrca** [OpenRCT2/OpenMusic#1]
They way we are planning on doing content for OpenRCT2 is to store music in the same way as rides and scenery. So each music track will be a custom object that is referenced by a park. This means that any custom music created (different songs from the original game) will be like custom rides and just hosted as independent entries on the online content database.

Therefore I think that it should be organised as follows:
* OpenMusic - this repository - contains only the original game music remastered.
* CustomMusic (any name you like, different repository) - your new music

You can then have a CI to render the music tracks and bundle them into our object format (yet to be designed) and uploaded to the content server (yet to exist).

**Comment by ccoors** [comment]
Fair enough, so I'll get to work on the new versions of the original soundtrack for now.

**Comment by Ruedii** [comment]
I think Custom Music should be a sub directory of the Open Music repository. 

I think we should also see about getting the files all formatted to a standard set of "Work in Progess" format.  We need to find the most raw formats that would work.

I noticed most of the music-roll type music is in lillypond.  While great for initial editing and typesetting, lillypond is very limited for post-processing and effects, so I would recommend Extended Midi format instead.    We should keep the original lilipond source files, though.  They are important base components for anyone who wants to start from scratch.  

I'd also recommend seeing if we can get a Jack advanced synth setup instead of our current base soft-synth. This would allow us far more choices of types of synthesizers, including a nice basic soft-synth where applicable.

**Comment by karst** [comment]
I have just created a README.md file in #13 

I think the following idea is the smartest way to go about it for OpenRCT2:  
As most scores included in RCT2 would still be protected under it's own copyright, we should not risk remastering the original tunes. I personally think it's best to deviate from the original tunes, so to create completely new tunes with the same theme. This way OpenRCT2 has it's own separate soundtrack from RCT2.  
When OpenRCT2 version 1.0.0 (the full standalone version of OpenRCT2) releases, you still have the option to load in RCT2's original assets, to use them as a "texture pack". 
This would be different for OpenMusic though. As you could use both soundtracks simultaneously. If someone loads in RCT2, the RCT2 soundtrack would come available.

What if someone doesn't have RCT2 installed, but they are loading a park from someone who does? Then these tunes will fall back on its OpenRCT2 equivalent.


I hope you guys like this idea. I personally think it's the best way of doing it.

**Comment by IntelOrca** [comment]
There will be three asset packs:
* `rct2.music` - All the music from RCT2.
* `openrct2.music.cover` - Cover of non-copyright music, e.g. merry-go-round, pirate theme, ragtime, organ etc.
* `openrct2.music.alternative` - Replaces copyright music with an OpenRCT2 official music object equivalent, e.g. space -> galaxy.

So you can reorder the asset packs to choose your priority as you see fit.
If you just want to use galaxy style in your park, you can add `openrct2.music.galaxy` to your park.

**Comment by karst** [comment]
Okay! That sounds fair! So I could add a column to the replacements/README.md that says type. And update the header description to add

> There will be three asset packs:
> 
> * `rct2.music` - All the music from RCT2.
> * `openrct2.music.cover` - Cover of non-copyright music, e.g. merry-go-round, pirate theme, ragtime, organ etc.
> * `openrct2.music.alternative` - Replaces copyright music with an OpenRCT2 official music object equivalent, e.g. space -> galaxy.

**Comment by Nornec** [comment]
Following as a music composer. If ever styles are needing re-envisioned or something new is warranted, I'd like to contribute. I'm new to learning of the projects existence so I don't know if the CI or content server exist yet but momentum from this year seems promising. In reading through the repo i also realized that there's no "sound file" per se but an actual .ly format. Never worked with this before, interesting.

**Comment by karst** [comment]
IntelOrca has been testing asset packs a few months back, but it's not yet completed.  I don't know if you are on the discord server, but we have a specific channel for it in there. More music would always be good 😊

**Comment by Nornec** [comment]
Great! Thank you. Just joined up.
